eis/vocabs/readme.md

```diff
@@ -0,0 +1,19 @@
+Repository for redirecting to videogame extension vocabularies
+===================
+
+Repository created to store the redirections of the the vocabulary for AutomationML
+
+At the moment we have three games extensions:
+
+/eis/vocabs/aml: 
+
+Html: https://github.com/i40-Tools/vocabularies/tree/master/AutomationML/amlDocs/
```

Comment:
Link is fine if that's what you want.  Wrong?  The "Widico ... 3 folders" block is munged into a paragraph.  You probably want to have triple backticks around it or indent it like it's code so it's formatted properly via markdown.

Hadn't heard of Manchester Syntax until just now.  Probably make a similar rule like the owl and turtle ones above except with MIME type and extension from here:
https://www.w3.org/TR/owl2-manchester-syntax/#Appendix:_Internet_Media_Type.2C_File_Extension_and_Macintosh_File_Type